demos/benchmark/lv_demo_benchmark.h

```diff
@@ -25,6 +25,34 @@ extern "C" {
  *      TYPEDEFS
  **********************/
 
+typedef struct {
+    const char * name;
+    void (*create_cb)(void);
+    uint32_t scene_time;
+    uint32_t cpu_avg_usage;
+    uint32_t fps_avg;
+    uint32_t render_avg_time;
+    uint32_t flush_avg_time;
+    uint32_t measurement_cnt;
+} lv_benchmark_scene_dsc_t;
+
+typedef struct {
+    /*
+     * List of scenes
+     * `create_cb` == NULL indicates the last stored scene
+     * Must not be free'd
```

Comment:
This is great documentation but I didn't understand it initially the first time a week ago and also today reading it again. Could it say:

```suggestion
     * The last scene in this array of scenes is terminated by a
     * sentinel scene that has `create_cb` == NULL.
     * Must not be free'd
```

Reply:
That's definitely better !